content/en/docs/tasks/security/authorization/authz-dry-run/index.md

```diff
@@ -0,0 +1,182 @@
+1. Run the following command to check the dry-run results in the proxy log. Note this requires you have enabled the
+   debug logging in the proxy. Try to send some more requests if you do not see the expected log.
+
+    {{< text bash >}}
+    $ kubectl logs "$(kubectl get pod -l app=httpbin -n foo -o jsonpath={.items..metadata.name})" -c istio-proxy -n foo  | grep shadow
+    2021-04-20T02:33:16.294417Z debug envoy rbac shadow denied, matched policy ns[foo]-policy[deny-path-headers]-rule[0]
+    {{< /text >}}
+
+1. The log `shadow denied, matched policy ns[foo]-policy[deny-path-headers]-rule[0]` means the request is rejected
+   by the dry-run policy `deny-path-headers` in the namespace `foo`.
+
+## Check dry-run result in metric using Prometheus
+
+1. Open the Prometheus dashboard with the following command:
+
+    {{< text bash >}}
+    $ istioctl dashboard prometheus
+    {{< /text >}}
+
+1. In the Prometheus dashboard, search for the metric `envoy_http_inbound_0_0_0_0_80_rbac{authz_dry_run_result!=""}`. The
+   following is an example metric output (you might find different values of these metrics):
+
+    {{< text plain >}}
+    envoy_http_inbound_0_0_0_0_80_rbac{app="httpbin",authz_dry_run_action="deny",authz_dry_run_result="allowed",instance="10.44.1.11:15020",istio_io_rev="default",job="kubernetes-pods",kubernetes_namespace="foo",kubernetes_pod_name="httpbin-74fb669cc6-95qm8",pod_template_hash="74fb669cc6",security_istio_io_tlsMode="istio",service_istio_io_canonical_name="httpbin",service_istio_io_canonical_revision="v1",version="v1"} 0
+    envoy_http_inbound_0_0_0_0_80_rbac{app="httpbin",authz_dry_run_action="deny",authz_dry_run_result="denied",instance="10.44.1.11:15020",istio_io_rev="default",job="kubernetes-pods",kubernetes_namespace="foo",kubernetes_pod_name="httpbin-74fb669cc6-95qm8",pod_template_hash="74fb669cc6",security_istio_io_tlsMode="istio",service_istio_io_canonical_name="httpbin",service_istio_io_canonical_revision="v1",version="v1"}  1
+    {{< /text >}}
+
+1. The metric `envoy_http_inbound_0_0_0_0_80_rbac{authz_dry_run_result="denied"}` has value `1` (you might find different
+   value depending on how many requests you have sent. It's expected as long as the value is greater than 0).
+   This means the dry-run policy applied to the `httpbin` workload on port `80` matched one request. The policy would
+   reject the request once if it was not in dry-run mode.
+
+## Check dry-run result in tracing using Zipkin
+
+1. Open the Zipkin dashboard with the following command:
+
+    {{< text bash >}}
+    $ istioctl dashboard zipkin
+    {{< /text >}}
+
+1. Find the trace result for the request from `sleep` to `httpbin`. Make sure you have set the sampling rate to 100 so
+   that you can quickly reproduce the trace result in this task. Try to send some more requests if you do see the trace
+   result.
+
+1. In the trace result, you should find the following custom tags indicating the request is rejected by the dry-run policy
```

Comment:
will be (what used below)

or "would be rejected"? since the actual request is not actually rejected.

Reply:
I thought about this but feel it seems better to directly describe the dry-run policy result